go/common/cbor/codec.go

```diff
@@ -21,21 +32,25 @@ var (
 			Name: "oasis_codec_size",
 			Help: "CBOR codec message size (bytes).",
 		},
-		[]string{"call"},
+		[]string{"call", "module"},
 	)
 
 	codecCollectors = []prometheus.Collector{
 		codecValueSize,
 	}
 
-	labelRead  = prometheus.Labels{"call": "read"}
-	labelWrite = prometheus.Labels{"call": "write"}
-
 	metricsOnce sync.Once
 )
 
+// MessageWrapper is a wrapper for MessageReader or MessageWriter.
+type MessageWrapper struct {
```

Comment:
All of this just for a single common string field? Consider just copying the field, it will simplify the code.